src/main/java/frc/robot/HardwareMap.java

```diff
@@ -12,41 +12,43 @@
 
     // either construct each subclass seperately in robotcontainer,
     // OR create fields for each subclass in the constructor for hardwaremap
```

Comment:
These comments are no longer necessary since we've decided how to use hardwaremap. should probably be replaced with actual documentation explaining the conventions we use.